docker/entrypoint.sh

```diff
@@ -50,12 +50,9 @@ else:
     printf "%s" "${PY_CREATE_SU_SCRIPT}" | python3 /tram/src/tram/manage.py shell
 fi
 
-# Run ML processes
-python3 /tram/src/tram/manage.py attackdata load
-python3 /tram/src/tram/manage.py pipeline load-training-data
-python3 /tram/src/tram/manage.py pipeline train --model nb
-python3 /tram/src/tram/manage.py pipeline train --model logreg
 nohup python3 /tram/src/tram/manage.py pipeline run --model logreg &
 
 # Run Django on port 8000
-python3 /tram/src/tram/manage.py runserver 0.0.0.0:8000
+# python3 /tram/src/tram/manage.py runserver 0.0.0.0:8000
+
+gunicorn tram.asgi:application -k uvicorn.workers.UvicornWorker -b 0.0.0.0:8000
```

Comment:
Would it be better to keep the dependencies slim and use gunicorn only? Or is there some added benefit to using uvicorn with gunicorn?

Reply:
I went back and forth. Uvicorn adds the async support to django, so I did it out of habit and figured why not. I could go either way though, no real preference. Thoughts?

Reply:
Might be simpler to use just one instead of two

Reply:
yea, no async code in the app, removed